src/components/group/GroupForm.js

```diff
@@ -128,7 +150,7 @@ export default class GroupForm extends Component {
           {` ${creation ? "Create" : "Update"} group`}
         </button>
         {" or "}
-        <Link to="/">Cancel</Link>
+        <AdminLink name="home" params={{}}>Cancel</AdminLink>
```

Comment:
Side note: we could definitely allow the `params` prop to not being passed here.